File: Desafio #01 - Numero palindromo/DART/numeroPalindromoCristhianMoreno97.dart
bool isPalindrome(BigInt number) {
  String numberStr = number.toString();
  return numberStr == numberStr.split('').reversed.join('');
}

void main() {
  print(isPalindrome(BigInt.from(121)));
  print(isPalindrome(BigInt.from(1212)));
  print(isPalindrome(BigInt.from(1211)));
  print(isPalindrome(BigInt.from(12121)));
  print(isPalindrome(BigInt.from(1200021)));
  print(isPalindrome(BigInt.parse('120002112000211200021120002112000211200021')));
  print(isPalindrome(BigInt.parse('1200021120002112000211200021120002112000211')));
}

File: Desafio #01 - Numero palindromo/JAVASCRIPT/numeroPalindromoCristhianMoreno97.js
function isPalindrome(number) {
    const numberStr = number.toString()
    return numberStr == numberStr.split('').reverse().join('')
}

console.log(isPalindrome(121))
console.log(isPalindrome(1212))
console.log(isPalindrome(1211))
console.log(isPalindrome(12121))
console.log(isPalindrome('121121'))
console.log(isPalindrome('121211212112121121211212112121121211212112121'))
console.log(isPalindrome('121211212112121121211212112121121211212112122'))